lib/hacluster.pm

```diff
@@ -1813,7 +1813,11 @@ sub crm_resource_meta_set {
         croak("Mandatory argument '$arg' missing.") unless $arg;
     }
     my $action = $args{argument_value} ? 'set' : 'delete';
-    assert_script_run("crm resource meta $args{resource} $action $args{meta_argument}");
+    my $cmd = "crm resource meta $args{resource} $action $args{meta_argument}";
```

Comment:
Is it an API change?

Code before:
- `argument_value` value was not used. It looks like it was used only to switch between `set` and `delete` mode. So seems like you can use it like `crm_resource_meta_set(..., argument_value => 1, ...)`

Code changed in this PR
- seems like presence is used to toggle between `set` and `delete` like before
- but in case is present, value is also appended to the command.

Could you please explain the API change in the function documentation at line 1804

I also noticed it is a relatively new API introduced in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21954/ and `argument_value` was not used before.

Reply:
It's a bugfix. If you check original function it would not pass any value to the `set` command, which does not work. 
Previously we would only use `delete` command so it was not found. 
The description is still accurate.